packages/cozy-procedures/src/components/Overview.jsx

```diff
@@ -12,32 +16,115 @@ import {
 } from 'cozy-ui/transpiled/react'
 import InlineCard from 'cozy-ui/transpiled/react/InlineCard'
 
+import Topbar from './Topbar'
+
 class Overview extends React.Component {
+  realtime = null
```

Comment:
do we need it?

Reply:
I guess that not really since if it is `undefined`, `if (!this.realtime) {` will be `true` too but I prefer that it is explicit, don't you?